src/app/util/Util.jsx

```diff
@@ -79,6 +79,20 @@ export const checkFeatureFlagActivated = (featureFlagList, componentStateObject)
   });
 };
 
+/**
+ * truncateStringOnWhitespace(str, maxLength)
+ * Return a version of the string shortened to the provided maxLength param. This includes
+ * the three characters for the ellipsis that is appended. If the string is shorter than the
+ * max length it is returned as is.
+ * @param {string} str - The string to be shortened (or returned without change)
+ * @param {int} maxLength - The maximum length of the returned string to be applied
+ */
+export const truncateStringOnWhitespace = (str, maxLength) => {
+  if (str.length < maxLength) { return str; }
+  const truncArray = str.substr(0, maxLength - 3).split(/\s+/).slice(0, -1);
```

Comment:
There is an edge case here that seems incredibly rare, but ... what if the entire title is one long word?  (input errors, may be?). In this case the title would be "..." because I think slice would return "".  I think that's ok - the other way to do it would be to truncate not on whitespace.

Reply:
either way, let's add a test for this so that it's documented.

Reply:
I think in that instance we should truncate not on whitespace, otherwise it will look to the user that we don't have a title, which is worse than ugly truncation I think.

But will definitely add a test